united.py

```diff
@@ -1,25 +1,51 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
+import calendar
 import csv
 from datetime import datetime
 import os
 import argparse
+from wsgiref import headers
+from xml.etree import ElementTree
 
-from flask.ext.sqlalchemy import SQLAlchemy
+import re
+from flask_sqlalchemy import SQLAlchemy
```

Comment:
flask.ext.sqlalchemy gave me a error, Apparently, there is a new
distribution method, the
extension  code is no longer stored under flaskext

2016-04-17 9:32 GMT+03:00 Daniel Hershcovich <notifications@github.com>:

> In united.py
> <https://github.com/hasadna/anyway/pull/579#discussion_r59978982>:
>
> >
> > -from flask.ext.sqlalchemy import SQLAlchemy
> > +import re
> > +from flask_sqlalchemy import SQLAlchemy
>
> What's the difference between this and flask.ext.sqlalchemy?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hasadna/anyway/pull/579/files/2dd7573bd5bab0315ca68dc10db70ec63fcc8e09#r59978982>
>